packages/docs/page-config/ui-elements/skip-link/examples/Default.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <va-skip-link 
```

Comment:
This doesn't seem to work for me and I have an idea on how we can document this component without too much trouble.

* Let's make it work for docs globally.
* Then on this page - just note: "To test this component - click `Esc`, then press `Tab` and you should be able to see a button that would get you straight to content on `Enter`. That's perfect for users who rely on keyboard navigation." (you can improve text with chatgpt if you want).
* And we can remove example.